src/common/src/log.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::num::NonZeroU32;
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+use governor::Quota;
+
+type RateLimiter = governor::RateLimiter<
+    governor::state::NotKeyed,
```

Comment:
Should we partition by source ids? Otherwise, if there're a lot of sources running, 10 logs per minute could be quite limited.

Reply:
Technically, Yes, but I prefer to keep it simple now ☺️ As it's randomly sampled, it's very likely that users/we can get the valuable information about every sources.